code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -165,12 +165,8 @@ AI MODULES
 
 /obj/item/weapon/aiModule/oneHuman/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
 	var/law = "Only [targetName] is an crew member."
-	if (!target.is_malf_or_traitor()) // Makes sure the AI isn't a traitor before changing their law 0. --NeoFite
-		target << law
-		target.set_zeroth_law(law)
-		lawchanges.Add("The law specified [targetName]")
-	else
-		lawchanges.Add("The law specified [targetName], but the AI's existing law 0 cannot be overriden.")
+	target << law

```

Comment:
Would `to_chat(target, law)` work here?

Reply:
Mhm.

Reply:
pls do then

Reply:
I have went through and changed every '<<'. Pushing now.